maintain_pim/first_aid_kit/index.rst

```diff
@@ -104,9 +104,9 @@ Step 9: did you consider the volume of your catalog?
 
 Each catalog is unique. How many channels, locales, attributes, families, categories, products and users do you have? These values may had increased drastically since you began using your PIM.
 
-Most of this information can be found on the PIM system information screen (under System > System information).
+Most of this information can be found on the PIM under the system information screen (under System > System information). Since 2.3 version, you have a dedicated page available under Activity menu, called Catalog Volume Monitoring, see our `_Help Center article <https://help.akeneo.com/pim/v3/articles/catalog-volume-monitoring.html#a-dedicated-screen>`__ for more information about this page and how to read it. 
 
-Once the calculations made, please have a look at our :doc:`/maintain_pim/scalability_guide/index` and :doc:`/technical_architecture/performances_guide/index`.
+You can also read this Medium article `_Because your product catalog typology matters <https://medium.com/akeneo-labs/because-your-product-catalog-typology-matters-e1a9af4c33e0>`__ to see the impacts of your catalog modeling on your PIM performances.
```

Comment:
```suggestion
You can also read this Medium article `_Because your product catalog typology matters <https://medium.com/akeneo-labs/because-your-product-catalog-typology-matters-e1a9af4c33e0>`__ to see the impacts catalog modeling can have on PIM performances.
```